client/layout/index.jsx

```diff
@@ -40,16 +40,9 @@ Layout = React.createClass( {
 
 	_sitesPoller: null,
 
-	componentWillUpdate: function( nextProps, nextState ) {
+	componentWillUpdate: function( nextProps ) {
 		if ( this.props.section !== nextProps.section ) {
-			if ( nextProps.section === 'sites' ) {
-				setTimeout( function() {
-					if ( ! this.isMounted() || this._sitesPoller ) {
-						return;
-					}
-					this._sitesPoller = PollerPool.add( this.props.sites, 'fetchAvailableUpdates', { interval: 900000 } );
-				}.bind( this ), 0 );
-			} else {
+			if ( nextProps.section !== 'sites' ) {
 				this.removeSitesPoller();
```

Comment:
We still add the main 'sites' poller here: https://github.com/Automattic/wp-calypso/blob/update/jetpack-sites-dont-poll-updates-check/client/lib/sites-list/index.js#L16

I think this intend to remove all "pollers" related to sites if we are not navigating something under `/sites`